src/user/dto/update-user.dto.ts

```diff
@@ -1,4 +1,39 @@
 import { PartialType } from '@nestjs/mapped-types';
 import { CreateUserDto } from './create-user.dto';
+import {
+  IsEmail,
+  MaxLength,
+  MinLength,
+  Matches,
+  IsOptional,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
 
-export class UpdateUserDto extends PartialType(CreateUserDto) {}
+export class UpdateUserDto extends PartialType(CreateUserDto) {
+  @IsOptional()
+  @ApiProperty({ example: 'user@example.com' })
+  @IsEmail()
+  email: string;
+
+  @ApiProperty({
+    example: 'password123',
+    description:
```

Comment:
Passport must be at least 6 characters or more than 6 symbols?

Reply:
usually 'characters' no 'symbols'. about the length: at least 6 and no more than 20 

`@MinLength(6, { message: 'Password must be more than 6 symbols!' })
  @MaxLength(20, { message: 'Password must be at most 20 characters long!' })`